test/test_shape.cpp

```diff
@@ -27,7 +27,6 @@ TEST_F(ShapeTest, Contains)
     ASSERT_TRUE(shape.contains(max));
 
     ASSERT_TRUE(shape.contains(side_center));
-    std::cout << "Side_center_triangle" << std::endl << std::endl << std::endl << std::endl << std::endl;
```

Comment:
? This seems unrelated to the lrf testcases.

Reply:
Yes indeed. But I found it and creating a PR to remove this single line might be too much